### DISCUSSION TITLE
最新版本的maa不能自己下载更新包

### DISCUSSION THREAD

**Discussion by djoiucxaoishe** [MaaAssistantArknights/MaaAssistantArknights#5761] [Q&A] [answered]
如题所示

**Comment by ABA2396** [comment]
v4.20.0版本服务器测速有问题，部分用户可能存在不更新解决不了，但不解决就不能自动更新问题.jpg

请遇到了《能检查到新版本，但无法下载》问题的用户手动更新以解决问题

能连上github的建议点 设置-关于我们-源码:Github 自行下载更新
连不上的点击 设置-关于我们- QQ群/QQ频道 下载安装包
或者等待正式版更新后在官网下载安装包

![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/99072975/8ea130f9-5e5b-4fd3-a3e5-d51b202630ed)

**Reply by djoiucxaoishe** [reply, marked as answer]
好的谢谢，辛苦了
